geckodriver.log

```diff
@@ -0,0 +1,112 @@
+1588451853848	mozrunner::runner	INFO	Running command: "/Applications/Firefox.app/Contents/MacOS/firefox-bin" "-marionette" "-foreground" "-no-remote" "-profile" "/var/folders/jc/_d63k96j10l1hc8rrpdfxhn40000gn/T/rust_mozprofileTp33P4"
```

Comment:
Is this necessary to commit? If not, we may want to delete this and add `geckodriver.log` to the `.gitignore`

Reply:
Ha, somehow I missed that file altogether! Agree this should be in `.gitignore`.